content/en/internal/configuration-database/access/_index.md

```diff
@@ -1,64 +1,65 @@
-- use an [X.509 digital certificate installed in the
-  browser](#using-an-x509-digital-certificate), delivered by
-  one of the recognised EU-Grid-PMA Certification Authorities ;
-
-Users can access the system as soon as they are authenticated however they will
-only be able to update information based on their roles.
+Users can access the system as soon as they are authenticated. However,
+they will only be able to update information based on their roles.
 More information about roles and associated permission is available in the
-[Users and
-roles](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Users_and_roles)
-section.
+[_Users and roles_](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Users_and_roles)
+section of the documentation.
 
-All roles applications need to be validated by parent roles or administrators.
-Once this is done, you can access/modify relevant information according to the
-role you have been granted. You can learn more on roles and user accounts by
-reading the #Users and roles section of this documentation.
+Applications requesting a specific role have to be validated by parent roles
+or administrators. Once granted, users can access and/or modify relevant
+information, according to the roles granted to the user.
 
 ## Using institutional account via EGI Check-in
 
 In order to be able to access the GOCDB with their institutional account, users
 need to:
 
 1. Have their Identity Provider (IdP) federated in EGI Check-in (via
-   [eduGAIN](https://edugain.org/) or directly) ;
+   [eduGAIN](https://edugain.org/) or directly)
 1. Have created an [EGI Check-in
-   account](https://wiki.egi.eu/wiki/AAI_usage_guide) ;
+   account](https://wiki.egi.eu/wiki/AAI_usage_guide)
 
 {{% alert title="Important" color="warning" %}}
 In the case the user cannot use an IdP compliant with [REFEDS
 R&S](https://refeds.org/research-and-scholarship) and [REFEDS
 Sirtfi](https://refeds.org/sirtfi), the user will have to request joining a
-specific group. Using a compliant IdP is the preferable solution.
-{{% /alert %}}
+specific group, by performing the steps below. Using a compliant IdP is the
+preferable solution.
 
 1. User should ask to join the [GOCDB user
-   group](https://aai.egi.eu/registry/co_petitions/start/coef:41) ;
+   group](https://aai.egi.eu/registry/co_petitions/start/coef:41)
 1. The access request will be managed by the EGI Operations team.
+{{% /alert %}}
 
 ## Using an X.509 digital certificate
 
-Please note, GOCDB does not support single or double quotes in the certificate
-Distinguished Name (DN).
+To access the configuration database using a digital certificate, first obtain
+a certificate from one of the recognised EU-Grid-PMA Certification Authorities
+(CAs), and install it on your local machine.
 
-This DN is rejected by GOCDB because of the single quote:
+{{% alert title="Note" color="info" %}} EGI Check-in does not support single
```

Comment:
This is not EGI Check-in the problem here.

Reply:
Fixed